app/domain/rotation/master_rotator.rb

```diff
@@ -60,6 +60,7 @@ def initialize(avail_rotators:, facade:, secret_model:)
       def run
         rotator.rotate(@facade)
       rescue => e
+        raise e
```

Comment:
Reraising `e` doesn't look right here. It'll bypass the error handling below.

Reply:
good catch, i put that there during testing and forgot to remove.